README.md

```diff
@@ -1,12 +1,115 @@
 # Riax
-
-## Starting:
+A tutorial on how to cache files over several Riak Nodes.
+## Quick start:
   * Install dependencies with `mix deps.get`
-  * Start Phoenix endpoint with `make phoenix`, this will set up both a Phoenix and Riak Node.
   * Start the Riak Nodes with `make node1`, `make node2`, `make node3`, this will
     have to be done on separate terminals, or using something like Tmux.
   * Now, try to join the nodes using `:riak_core.join(node@host)` and use the Riax
     module as an API connecting to each Node.
-## Simple Example:
-- You can visit [`localhost:4003`](http://localhost:4003/) from your browser
-  to see a simple example running.
+## Riak Setup:
+  *  PLACEHOLDER: This is where the setup of Riak using the Elixir library
+     would be. This could be either a setup here, o a link to the library.
+     Either way, the tutorial would start with a working Riak key-value.
+     I'm thinking of providing it along with the VNode implementation, as
+     to no repeat the Erlang tutorial.
+## Tutorial:
+### Use Case:
```

Comment:
you need to improve the description of this use case, and probably move it up to the start of the tutorial.
you have to state it in terms of the specific tweet dataset, not in terms of serving csv files (the csv portion is an implementation detail)

Reply:
also you have to stress what's the part of the problem where riak gives you a good solution

Reply:
Yes, I like that, thanks for the feedback.